src/pydata_sphinx_theme/assets/styles/variables/_fonts.scss

```diff
@@ -4,23 +4,23 @@ html {
   */
 
   // base font size - applied at body/html level
-  --pst-font-size-base: 15px;
+  --pst-font-size-base: 1rem;
```

Comment:
Note that this changed default font size from 15px to 16px; this causes all text to be a bit bigger, cf. https://matplotlib.org/3.7.1 vs https://matplotlib.org/devdocs.

I didn't see any specific discussion on whether that change was intentional here?